queries/vala/highlights.scm

```diff
@@ -247,12 +236,14 @@
 ; New instance from Object
 
 (new_instance
-  ".new" @keyword
+  "new" @keyword
 )
 
 ; GObject construct
 
-"construct" @constructor
+(gobject_construct 
+  "construct" @constructor
```

Comment:
this should probably be a keyword. `@constructor` in other languages are often the functions/types called to construct an object.